bigframes/session/executor.py

```diff
@@ -323,8 +324,51 @@ def export_gbq(
             sql=sql,
             job_config=job_config,
         )
+        self._attach_timedelta_metadata(
+            query_job.destination, array_value, col_id_overrides
+        )
         return query_job
 
+    def _attach_timedelta_metadata(
+        self,
+        table_ref: bigquery.TableReference | None,
+        array_value: bigframes.core.ArrayValue,
+        col_id_overrides: Mapping[str, str],
+    ):
+        """Updates the descriptions of columns that are holding timedeltas values with a special tag."""
+        assert table_ref is not None
+
+        table = self.bqclient.get_table(table_ref)
+        table_def = table.to_api_repr()
+        for field in table_def["schema"]["fields"]:
```

Comment:
Do we need to be recursing into the sub fields if this is a RECORD / STRUCT so we can label those, too?

Or maybe leave that as a TODO?

Reply:
Also, I'm wondering about the cost of this loop. Some tables might have 10,000 columns. This will likely be pretty slow in Python.

Reply:
RECORD columns are complicated in the sense that it can contain both timedelta and integer columns, so our tag needs to have some kind of recursive structures too. Shall we wait for FRs from customers before evaluating the required effort?

Good call for the loop comment. I didn't realize it's a nested loop. Updated the code such that timedelta column labels are passed in as a set, and the lookup of each field costs O(1)